rbx_binary/src/deserializer.rs

```diff
@@ -77,6 +80,164 @@ pub(crate) enum InnerError {
 
     #[error("File referred to type ID {type_id}, which was not declared")]
     InvalidTypeId { type_id: u32 },
+
+    #[error("Invalid property data: CFrame property {type_name}.{prop_name} had an invalid orientation ID {id:02x}")]
+    BadCFrameOrientationId {
+        type_name: String,
+        prop_name: String,
+        id: u8,
+    },
+}
+
+// TODO potentially move this to a different file if/when we do the inverse for serializing
+pub(crate) fn special_case_to_rotation(id: u8) -> Option<Matrix3> {
+    match id {
+        0x02 => Some(Matrix3::new(
+            //02 -> (1, 0, 0) (0, 1, 0) (0, 0, 1)
```

Comment:
Do you think we need these comments? They look like they might be left over from when you were writing out the code from the cases.